src/Makefile.am

```diff
@@ -64,6 +62,11 @@ libopendht_la_SOURCES += dht_proxy_client.cpp
 nobase_include_HEADERS += ../include/opendht/dht_proxy_client.h ../include/opendht/dht_interface.h
 endif
 
+if ENABLE_INDEXATION
```

Comment:
OPENDHT_INDEXATION or change the previous add_definitions.

Reply:
this is an AM_CONDITIONAL not a define

Reply:
oh my bad... was in the CMakeLists...